backend/src/main/kotlin/app/ehrenamtskarte/backend/matomo/Matomo.kt

```diff
@@ -153,7 +150,7 @@ object Matomo {
         if (params.searchText === null && params.categoryIds === null) return
         sendTrackingRequest(
             projectConfig.matomo,
-            MatomoRequest.builder()
+            MatomoRequest.request()
                 .actionName(query)
                 .searchQuery((params.searchText ?: "") + "?categories=" + (params.categoryIds?.joinToString(",") ?: ""))
```

Comment:
Could you also split search query and search category, as this works now as intended? Would be awesome :)
```suggestion
     .searchCategory(params.categoryIds?.joinToString(","))
     .searchQuery(params.searchText ?: "")
```